Docker/build-images.ps1

```diff
@@ -104,7 +104,11 @@ param (
 
     [Parameter()]
     [string]
-    $ExtensionVersion = $env:EXTENSION_VERSION
+    $ExtensionVersion = $env:EXTENSION_VERSION,
+
+    [Parameter()]
+    [string]
+    $LatestStandardVersion = $env:CURRENT_STANDARD_VERSION
```

Comment:
Can you remove this $LatestStandardVersion  parameter and  you can update below code
 $stdVer = "-${{ $env:CURRENT_STANDARD_VERSION }}"
    if (${{ $env:CURRENT_STANDARD_VERSION }} -eq $StandardVersion) {
        $stdVer = ""
    }

Reply:
removed $LatestStandardVersion